Assets/UI/Panels/citypaneloverview.lua

```diff
@@ -1153,6 +1264,29 @@ end
 -- ===========================================================================
 function OnShutdown()
     LuaEvents.GameDebug_AddValue(RELOAD_CACHE_ID, "m_isShowingPanel", m_isShowingPanel);
+
+    LuaEvents.Tutorial_ResearchOpen.Remove(OnClose);
```

Comment:
matches the OnShutdown function in unmodified game, calling Remove on these things
I verified each of these removes corresponds with an "add"